docs/source/api/spawner.md

```diff
@@ -10,11 +10,12 @@
 
 ```{eval-rst}
 .. autoconfigurable:: Spawner
-   :members: options_from_form, poll, start, stop, get_args, get_env, get_state, template_namespace, format_string, create_certs, move_certs
+   :members:
 ```
 
 ### {class}`LocalProcessSpawner`
 
 ```{eval-rst}
 .. autoconfigurable:: LocalProcessSpawner
+   :members:
```

Comment:
Previews:
- This PR: https://jupyterhub--4253.org.readthedocs.build/en/4253/api/spawner.html#localprocessspawner
- Main branch: https://jupyterhub.readthedocs.io/en/latest/api/spawner.html#localprocessspawner